wiki/Manual.md

```diff
@@ -6339,43 +6337,48 @@ Here is an outline of exception inheritance hierarchy: https://github.com/ccxt/c
 |   |
 |   +---+ NotSupported
 |
-+---+ NetworkError (recoverable)
-    |
-    +---+ InvalidNonce
-    |
-    +---+ RequestTimeout
-    |
-    +---+ ExchangeNotAvailable
-    |   |
-    |   +---+ OnMaintenance
+|
++---+ OperationFailed (recoverable)
     |
-    +---+ DDoSProtection
+    +---+ NetworkError (recoverable)
         |
-        +---+ RateLimitExceeded
+        +---+ InvalidNonce
+        |
+        +---+ RequestTimeout
+        |
+        +---+ ExchangeNotAvailable
+        |   |
+        |   +---+ OnMaintenance
+        |
+        +---+ DDoSProtection
+            |
+            +---+ RateLimitExceeded
 ```
 
 The `BaseError` class is a generic root error class for all sorts of errors, including accessibility and request/response mismatch. If you don't need to catch any specific subclass of exceptions, you can just use `BaseError`, where all exception types are being caught.
 
-From `BaseError` there are derived two different families of the error hierarchy:
-
-- `NetworkError`
-- `ExchangeError`
+From `BaseError` derives two different families of errors: `OperationFailed` and `ExchangeError` (they also have their specific sub-types, as explained below).
 
-They also have more specific sub-type exceptions, as explained below.
+### OperationFailed
+<a name="NetworkError" id="NetworkError"></a>
 
-### NetworkError
+An `OperationFailed` happens when user sends **correctly constructed & valid request** to exchange, but an non-deterministic problem occurred. In suchcases, you might consider retrying after some time.
```

Comment:
```suggestion
An `OperationFailed` happens when user sends **correctly constructed & valid request** to exchange, but an non-deterministic problem occurred. In such cases, you might consider retrying after some time.
```

Reply:
yeah yeah noticed it also, already fixed